src/test/java/com/vackosar/gitflowincrementalbuild/control/DifferentFilesTest.java

```diff
@@ -215,6 +221,54 @@ public void fetchNonExistent() throws Exception {
         assertCommitExists(FETCH_FILE, localGit);
     }
 
+    @Test
+    public void noLocalRepo() throws Exception {
```

Comment:
I'd say that this is already covered by [`GitProviderTest`](https://github.com/vackosar/gitflow-incremental-builder/blob/version/3.11.2/src/test/java/com/vackosar/gitflowincrementalbuild/jgit/GitProviderTest.java#L72-L81).

Reply:
removed.